src/main/java/org/folio/rest/impl/base/AbstractEmail.java

```diff
@@ -135,21 +139,25 @@ protected Future<EmailEntity> processEmail(EmailEntity email, SmtpConfiguration
 
   protected EmailEntity handleSuccess(EmailEntity email) {
     String message = format(SUCCESS_SEND_EMAIL, join(",", email.getTo()));
-    logger.info(message);
+    logger.debug("handleSuccess:: parameter email: {}", () -> logAsJson(email));
```

Comment:
Logging of the method name and parameters should be the first line in the method